src/main/drivers/serial_uart_stm32f4xx.c

```diff
@@ -264,10 +264,7 @@ static void handleUsartTxDma(uartPort_t *s)
 {
     DMA_Cmd(s->txDMAStream, DISABLE);
```

Comment:
Another transfer may have been started in `uartWrite` before this interrupt was served ..